src/ApiService/ApiService/onefuzzlib/notifications/Ado.cs

```diff
@@ -529,6 +529,7 @@ public sealed class AdoConnector {
             // I haven't found any other combinations where System.Reason could be duplicate but just to be safe
             // we're explicitly _not_ checking the state of the work item to determine if it's duplicate
             return wi.Fields.ContainsKey("System.Reason") && string.Equals(wi.Fields["System.Reason"].ToString(), "Duplicate")
+            || wi.Fields.ContainsKey("Microsoft.VSTS.Common.ResolvedReason") && string.Equals(wi.Fields["Microsoft.VSTS.Common.ResolvedReason"].ToString(), "Duplicate")
```

Comment:
I can't remember if the .Equals is case sensitive or not.

Reply:
And we're sure these are fields that would apply regardless of the ADO org?

Reply:
It is case sensitive by default, I could make those insensitive.

It's ok if the fields aren't present in the work item, it'll just evaluate to false. This is just a quick fix until we can implement the proper fix described here: https://github.com/microsoft/onefuzz/issues/3384

Reply:
Made the comparisons case insensitive